examples/window.rs

```diff
@@ -0,0 +1,15 @@
+#![windows_subsystem = "windows"]
+
+#[macro_use]
+extern crate human_panic;
+
+fn main() {
+  setup_panic!(Metadata {
+    name: "Human Panic Window Example".into(),
+    version: env!("CARGO_PKG_VERSION").into(),
+    authors: "ルカス".into(),
+    homepage: "https://github.com/rust-clique/human-panic/issues".into(),
+    create_window: true,
```

Comment:
Also something I'm noticing: if we have an `gui` compile feature we don't need to pass this anymore. The feature becomes an opt-in at the dependency level which is much neater IMO.